docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -669,18 +695,19 @@ stage. Jobs and SQL are not color coordinated.
 Below we will aggregate the task level metrics at different levels to do some analysis such as detecting possible shuffle skew.
 
 - Job + Stage level aggregated task metrics:
-```
+
 ### B. Analysis ###
```

Comment:
For "### B. Analysis ###", same as before, it is command output. We can either remove it or keep it in the code block.

Reply:
Moved those back to the code block.